src/main/groovy/org/mockito/release/internal/gradle/BaseJavaLibraryPlugin.java

```diff
@@ -114,8 +97,12 @@ public void execute(VersioningPlugin versioningPlugin) {
                 //The generate pom task is dynamically created by Gradle and we can only access it during execution
                 t.comparePom(POM_TASK);
 
-                t.setProjectGroup(project.getGroup().toString());
-                t.setProjectName(project.getName());
+                LazyConfiguration.lazyConfiguration(t, new Runnable() {
```

Comment:
Let's use deferred configuration, it's more fitting this case. For more info check the docs on lazy vs deferred configuration.